### PR TITLE
LPD-36230 - Strip out pathContext before providing as portal-header urls

### PR DESCRIPTION
**Pull request by bryceosterhaus**

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:relevant

File: modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/portlet/CustomElementCETPortlet.java [hunk 1]
```diff
@@ -162,6 +166,10 @@ private String[] _prepareURLs(long lastModified, String[] urls) {
 			if (!urls[i].startsWith("module:")) {
 				urls[i] = "nocombo:" + urls[i];
 			}
+
```
Comment (izaera) [inline modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/portlet/CustomElementCETPortlet.java:169]: The `if` above is looking for `module:` and/or adding `nocombo:` depending on what comes in `urls[i]`. That makes me think that your new `if` will fail under some conditions maybe.

Also: why "not startsWith"? Shouldn't it be "startsWith"? :thinking:

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Pull request closed without merging**